examples/elasticache/v1beta1/serverlesscache.yaml

```
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: elasticache.aws.upbound.io/v1beta1
+kind: ServerlessCache
+metadata:
+  annotations:
+    meta.upbound.io/example-id: elasticache/v1beta1/serverlesscache
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    cacheUsageLimits:
+    - dataStorage:
+      - maximum: 10
+        unit: GB
+      ecpuPerSecond:
+      - maximum: 5000
+    description: Test Server
+    engine: memcached
+    majorEngineVersion: "1.6"
+    region: us-west-1
+    securityGroupIdSelector:
+      matchLabels: 
+        testing.upbound.io/example-name: example
+    subnetIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+---
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SecurityGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: elasticache/v1beta1/serverlesscache
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
```

Also, it would be great to change the following names.

```suggestion
  name: example-sg-serverlesscache
```